Portfolio/index.css

```diff
@@ -1 +1,48 @@
-/* Write your code below for PORTFOLIO */
+<html>
+<head>
+<title>Portfolio</title>
+</head>
+<h2>Kumar</h2>
+<body>
+<ul>
+<li>ABOUT</li>
+<li>SKILLS</li>
+<li>PROJECTS</li>
+<li>REVIEWS</li>
+</ul>
+<button>CONTACT ME</button><br>
+
+<section> <h1>Aditya Kumar<h1><br>
```

Comment:
The image source should be a valid filename or URL. Update the src attribute with the correct filename or URL.